network-api/networkapi/templates/fragments/buyersguide/pni_newsletter_box.html

```diff
@@ -1,6 +1,6 @@
 {% load wagtailcore_tags i18n static %}
 
-<div class="tw-relative tw-py-5 tw-px-5 tw-rounded-lg medium:tw-px-7 medium:tw-py-6 tw-bg-gradient-to-r tw-from-purple-05 tw-to-blue-05">
+<div id="buyersguide_newsletter_box" class="tw-hidden tw-relative tw-py-5 tw-px-5 tw-rounded-lg medium:tw-px-7 medium:tw-py-6 tw-bg-gradient-to-r tw-from-purple-05 tw-to-blue-05">
```

Comment:
What do you think about using `-` instead of `_` in the HTML `id` attribute? That seems like the more common style in the code base.

Reply:
Why was this resolved?

Reply:
I must have missed that! I thought I addressed that on a previous commit. It is added now @tbrlpld